notebooks/notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>Since this plot represents the spatial distribution of errors, I suggest generating a map. You could explore the use of cartopy.</p><p>See:</p><p>https://docs.xarray.dev/en/stable/user-guide/plotting.html#maps</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/repro-challenge-team-3/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>Thank you! We'll consider this for the final submission! :)</p>

Reply:
closing conversation, resolved